webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/list/ActivityStreamList.vue

```diff
@@ -111,6 +111,10 @@ export default {
     },
   },
   created() {
+    this.streamFilter = localStorage.getItem('activity-stream-stored-filter');
+    if (this.streamFilter) {
+      document.dispatchEvent(new CustomEvent('activity-stream-stored-filter-applied', {detail: this.streamFilter}));
```

Comment:
We use `document.addEventListener` to allow dispatching events outside the application. When the event is about an internale mechanism of the application, it's better to use `this.$root.$emit` and `this.$root.$on` to simplify event management.

Reply:
Done!